plugins/bkpr/account.c

```diff
@@ -1,17 +1,18 @@
 #include "config.h"
 
 #include <ccan/str/str.h>
+#include <ccan/tal/str/str.h>
 #include <common/coin_mvt.h>
 #include <common/node_id.h>
 #include <plugins/bkpr/account.h>
 
 struct account *new_account(const tal_t *ctx,
-			    const char *name STEALS,
+			    const char *name,
```

Comment:
Technically, this is now TAKES (since tal_strdup() can take).

Reply:
i just removed the thing b/c we don't ever call this with `take`... (and it does strdup)

Reply:
But it *would* take, because tal_strdup() does this...